README.md

```diff
@@ -128,7 +171,7 @@ Your Answer:
 
 Select all that apply:
 ```
-[] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
-[] Synchronous code runs all at the same time, asynchronous code runs completely randomly
-[] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
+[x] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
+[?] Synchronous code runs all at the same time, asynchronous code runs completely randomly " I wouldn't say 'completely randomly...' async code runs when the parameters of its call back/event has been met / enacted. There is a method to the madness (no pun intended... ok maybe a small pun intended)"
+[x] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
 ```
```

Comment:
The correct answer is the last one. Asynchronous code isn't inherently random, although you could set up your code to act as such.